### PR TITLE
Feature/get files method

### PR DESCRIPTION
**Pull request by EugeneZverev**
In script docManager.js added method getFilesInFolderInfo(). First, we get the files using the getStoredFiles() method, then iterate over them. An object fileObject is created with information about the file. Each object is added to an array, which we send to the client in script app.js.
In the app.js script, a app.get("/files", function(req, res) request handler has been added, which receives information about files using the getFilesInFolderInfo() method and sends it back to the client.

### CODE REVIEW

File: web/documentserver-example/nodejs/helpers/docManager.js [hunk 1]
```diff
@@ -369,4 +369,51 @@ docManager.cleanFolderRecursive = function (folder, me) {
...
+                        contentLength: `${(fileSystem.lstatSync(curFilePath).size/1024).toFixed(2)} KB`,
+                        pureContentLength: fileSystem.lstatSync(curFilePath).size, 
+                        title: curFile,
+                        updated: fileSystem.lstatSync(curFilePath).mtime
+                    };
+                    responseArray.push(fileObj);
+                }
+            })
+        }
+    })*/
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/helpers/docManager.js:415]: remove old code

File: web/documentserver-example/nodejs/helpers/docManager.js [hunk 2]
```diff
@@ -369,4 +369,51 @@ docManager.cleanFolderRecursive = function (folder, me) {
...
+    //new logic
+    const userAddress = docManager.curUserHostAddress();
+    const directory = path.join(docManager.dir, userAddress);
+    const filesInFolder = this.getStoredFiles();
+    var responseArray = [];
+    filesInFolder.forEach((file)=>{
+        const stats = fileSystem.lstatSync(path.join(directory, file.name));
+        const fileObject = {
+            version: file.version,
+            fileStatus: Number(!file.canEdit), //0, if the file can be edited, and 1 if not
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/helpers/docManager.js:382]: What is the parameter `fileStatus` and why is it numerical?

File: web/documentserver-example/nodejs/app.js [hunk 3]
```diff
@@ -651,6 +651,22 @@ app.get("/editor", function (req, res) {
     }
 });
 
+app.get("/files", function(req, res){
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/app.js:654]: Who will use this request? Why add it?

File: web/documentserver-example/nodejs/app.js [hunk 4]
```diff
@@ -651,6 +651,22 @@ app.get("/editor", function (req, res) {
...
+            count: filesInFolderInfo.length,
+            response: filesInFolderInfo
+        };
+        res.write(JSON.stringify(responseObj));
+    } catch (ex){
+        console.log(ex);
+        res.write("Server error");
+    }
+    res.end();
+})
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/app.js:668]: Please move this code to the file request level
https://github.com/ONLYOFFICE/document-server-integration/blob/48c0bef1858b55184ea8128ede86198df52d0592/web/documentserver-example/nodejs/app.js#L291

File: web/documentserver-example/nodejs/helpers/docManager.js [hunk 5]
```diff
@@ -369,4 +369,51 @@ docManager.cleanFolderRecursive = function (folder, me) {
     }
 };
 
+docManager.getFilesInFolderInfo = function (){
+    //new logic
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/helpers/docManager.js:373]: remove

**Review by LinneyS — CHANGES_REQUESTED** [review]

File: web/documentserver-example/nodejs/helpers/docManager.js (hunk 2 above)
Comment (EugeneZverev) [reply web/documentserver-example/nodejs/helpers/docManager.js]: fileStatus is incorrect. I will need to fix this.

File: web/documentserver-example/nodejs/app.js (hunk 4 above)
Comment (EugeneZverev) [reply web/documentserver-example/nodejs/app.js:668]: I saw it, but did not know what to call it correctly

File: web/documentserver-example/nodejs/app.js (hunk 3 above)
Comment (EugeneZverev) [reply web/documentserver-example/nodejs/app.js:654]: I dont know. I didn't know how to correctly implement the logic.

File: web/documentserver-example/nodejs/helpers/docManager.js (hunk 1 above)
Comment (EugeneZverev) [reply web/documentserver-example/nodejs/helpers/docManager.js]: old code has been removed

File: web/documentserver-example/nodejs/helpers/docManager.js (hunk 5 above)
Comment (EugeneZverev) [reply web/documentserver-example/nodejs/helpers/docManager.js]: it removed

**Comment by LinneyS** [comment]
@hellonadya 
Please add a **GET `/files`** request to readme

**Pull request merged**